interface/forms/procedure_order/new.php

```diff
@@ -226,13 +231,27 @@ function getListOptions($list_id , $fieldnames=array('option_id', 'title', 'seq'
  padding-right:2px;
 }
 
+.form-inline
```

Comment:
In terms of best practice, would it also be best not to modify the styling of the bootstrap classes? And I am guessing there are lots of boostrap modifier classes that may get Sherwin what he needs here?
(note I'm still learning best way to use bootstrap)